api_tests/collection_submissions_actions/views/test_collection_submissions_actions_detail.py

```diff
@@ -0,0 +1,120 @@
+import pytest
+from api_tests.utils import UserRoles
+from osf_tests.factories import AuthUserFactory
+from osf_tests.factories import NodeFactory, CollectionFactory, CollectionProviderFactory
+
+from osf.migrations import update_provider_auth_groups
+from osf.models import CollectionSubmission
+
+GET_URL = '/v2/collection_submissions_actions/{}/'
+
+
+@pytest.fixture()
+def collection_provider():
+    collection_provider = CollectionProviderFactory()
+    update_provider_auth_groups()
+    return collection_provider
+
+
+@pytest.fixture()
+def node(collection_provider):
+    node = NodeFactory(is_public=True)
+    node.provider = collection_provider
```

Comment:
`provider` should only be a `RegistrationProvider`, this will be associated through the `CollectionSubmission`